backend/requirements.txt

```diff
@@ -107,4 +107,4 @@ wcwidth==0.2.5
 whitenoise==5.2.0
 wrapt==1.12.1
 xlrd==2.0.1
-xlwt==1.3.0
+xlwt==1.3.0
```

Comment:
I don't see the iteration_utilities in those requirements.
Also, why there's requirements.txt at root and this other one? I think there should be only one at root.

Reply:
I was put on the wrong path. I will fix it, thanks :)